robot_driver/README.md

```diff
@@ -0,0 +1,162 @@
+# Global Body Planner
```

Comment:
This is just meant as a template here?

Reply:
Looks like that got missed. I just updated it to make sure front-facing documentation is correct, we'll need to merge it back into devel though.